addons/mail/static/src/components/chat_window_header/chat_window_header.js

```diff
@@ -41,6 +41,22 @@ class ChatWindowHeader extends Component {
         return this.env.models['mail.chat_window'].get(this.props.chatWindowLocalId);
     }
 
+    /**
+     * @returns {string}
+     */
+    get shiftNextText() {
+        const text = this.env.messaging.locale.textDirection === 'rtl' ? 'Shift left' : 'Shift right';
+        return this.env._t(text);
```

Comment:
this is not how `_t` works, it's parsing the text inside it for translation, you can't use a variable